docs/docs/configuration/overview.md

```diff
@@ -192,6 +192,7 @@ Provider specific options can be found on their respective subpages.
 | flag: `--allow-query-semicolons`<br/>toml: `allow_query_semicolons`       | bool           | allow the use of semicolons in query args ([required for some legacy applications](https://github.com/golang/go/issues/25192))                                                                                                                                                                                                                                                                                                                                                                                        | `false`     |
 | flag: `--api-route`<br/>toml: `api_routes`                                | string \| list | return HTTP 401 instead of redirecting to authentication server if token is not valid. Format: path_regex                                                                                                                                                                                                                                                                                                                                                                                                             |             |
 | flag: `--authenticated-emails-file`<br/>toml: `authenticated_emails_file` | string         | authenticate against emails via file (one per line)                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |             |
+| flag: `--deny-invalid-bearer-tokens`<br/>toml: `deny_invalid_bearer_tokens`       | bool           | if `--skip-jwt-bearer-tokens` is set, if a request includes an invalid JWT (expired, malformed, missing required audiences, etc), respond with 403 instead of falling back to authorization code grant redirect | false       |
```

Comment:
I could see us reversing the default on this in a future release, with that in mind, if the name of this were reversed, so that the current default would be true, what would you call this?

Maybe something like `--fallback-to-login-flow` or something?

Reply:
I'm updating with `--bearer-token-login-fallback`, let me know if that sounds good (sure is harder to add a default true option!)